routing.rst

```diff
@@ -120,19 +120,21 @@ the ``BlogController``:
     .. code-block:: php
 
         // config/routes.php
+        namespace Symfony\Component\Routing\Loader\Configurator;
```

Comment:
I’m sorry if it’s a dumb question but why does it not use something like the `App\` namespace? It looks like it’s highjacking Symfony’s namespace.